src/Column/ColumnDefinitionBuilder.php

```diff
@@ -40,6 +41,10 @@ final class ColumnDefinitionBuilder extends AbstractColumnDefinitionBuilder
 
     public function build(ColumnInterface $column): string
     {
+        if ($column->isUnsigned()) {
+            throw new NotSupportedException('The "unsigned" attribute is not supported by PostgreSQL.');
+        }
```

Comment:
I'd just ignore this. Or as an option we can add `CHECK (column >= 0)`

Reply:
Unsigned columns often usage for big values. Exception will alert the user that this is not available.